File: POO/PHP/UberPool.php
<?php 
require_once('Car.php');

class UberPool extends Car{
    public $brand;
    public $model;

    public function __construct($license, $driver, $brand, $model){
        parent::__construct($license, $driver, $brand, $model);

        $this->brand = $brand;
        $this->model = $model;
    }

    public function setPassenger($passenger){
        if ($passenger == 6) {
            $this->passenger = $passenger;
        }else{
           echo("Se necesita minimo 6 pasajeros"); 
        }
    }

}

File: POO/Java/UberPool.java
class UberPool extends Car{
    Stirng brand;
    Stirng model;
    
    public UberPool(String license, Account driver, String brand, String model){
        super(license, driver);
        this.brand = brand;
        this.model = model;
    }

    public void setPassenger(Integer passenger){
        if(passenger == 4){
            super.setPassenger(passenger);
        }else{
            System.out.println("UberVan necesita 4 pasajeros");
        }
    }
}